syntax = "proto3";

message WeatherRequest {
    string location = 1;
}

message WeatherLocation {
    string location = 1;
    float temp = 2;
    string condition_text = 3;
    string condition_img = 4;
}

message WeatherResponse {
    repeated WeatherLocation location = 1; 
}

service WeatherGrpcService {
    rpc Weather (WeatherRequest) returns (WeatherResponse);
}